plugin/src/shared/utils/cli-utils.ts

```diff
@@ -118,4 +118,33 @@ export const selectApplication = async (tree: Tree, message: string) => {
       return entries.filter((entry) => entry.name.toLowerCase().includes(input.toLowerCase()));
     }
   });
-}
+};
+
+export const getLibrariesDetails = (tree: Tree) => Array.from(getProjects(tree))
+  .filter(([_, project]) => project.projectType === 'library')
+  .map(([name, project]) => ({ name, path: project.root }));
+
+export const selectLibrary = async (tree: Tree, message: string) => {
```

Comment:
Let's unite `selectLibrary` and `selectApplication` into one function with a parameter `projectType`